build/projects/Add-Ons (W1)/.AL-Go/settings.json

```diff
@@ -0,0 +1,36 @@
+{
+    "projectName": "Add-Ons (W1)",
+    "appFolders": [
+        "../../../src/Add-Ons/W1/*/App"
+    ],
+    "testFolders": [
+        "../../../src/Add-Ons/W1/*/Test"
+    ],
+    "buildModes": [
+        "Translated"
+    ],
+    "installOnlyReferencedApps": false,
+    "ConditionalSettings": [
+        {
+            "buildModes": [
+                "Clean"
+            ],
+            "settings": {
+                "doNotRunTests": true,
+                "useCompilerFolder": true,
```

Comment:
There are scripts overrides to deal with running tests, etc., but these settings suggest tests are only built and with a compiler folder, not a container.

Reply:
That is indeed the case for CLEAN mode. In order to run tests in CLEAN mode we'd need to run on a self-hosted runner. Otherwise, we can't publish the CLEAN base application. 

We need the script overrides for default/translated mode since we do run tests in those buildmodes.